Gemfile

```diff
@@ -215,8 +215,6 @@ end
 
 group :development, :test do
   gem "pry-rails"
-  gem "debugger", :platforms => [:mingw_19, :ruby_19]
-  gem 'debugger-xml', :platforms => [:mingw_19, :ruby_19]
   gem 'byebug', :platforms => [:mingw_20, :ruby_20, :ruby_21]
```

Comment:
:elephant: I think you can remove the `:platforms` constraint from `byebug` & friend so you don't have to tweak this every time you upgrade a minor version of ruby